docs/user-guide/projects/project-lists/README.md

```diff
@@ -94,14 +95,20 @@ Projects can be filtered in OpenProject. The default view will list all currentl
 
 **Archived projects** - returns all projects that are not active, of which you were a member or have the right to see.
 
+**Public project lists** - shows all project lists that are public, i.e. that were shared with everyone within an OpenProject instance.
+
 **My private project lists** - shows all the project lists that you have customized and saved.
 
+**Shared project lists** - shows all the project lists that were shared with you. 
```

Comment:
@MayaBerd here we will need to add "that were shared with you or you shared with others."